cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -250,10 +255,9 @@ func (mh *MessageHandle) ServeConn(hi hubio.CloudHubIO, info *model.HubInfo) {
 
```

Comment:
Because  not every time node connection establishment will call `ServeConn`, how about changing it to `ServeNode` , using ` mh.nodeConns.get(nodeName)` in this func instead of the parameter of `hi hubio.CloudHubIO` . Then the task of `OnRegister(better change to OnConnected)` is to change the dead broken connection to the new active one.

Reply:
make sense, done